api/vendor/github.com/openshift/assisted-service/models/cluster_create_params.go

```diff
@@ -42,6 +42,9 @@ type ClusterCreateParams struct {
 	// Cluster networks that are associated with this cluster.
 	ClusterNetworks []*ClusterNetwork `json:"cluster_networks"`
 
+	// The amount of control planes which should be part of the cluster in high availability 'Full' mode.
+	ControlPlaneCount *int64 `json:"control_plane_count,omitempty"`
```

Comment:
How come we're storing as `int64` instead of just `int`? Just curious because of all the conversions I see like this one https://github.com/openshift/assisted-service/pull/6917/files#diff-40883156aab921707c5a73dfa3b451bbd3accb4c0636d715d0a7768db8904b57R683

Reply:
This is generated by swagger as int64, so I thought it would result in less conversions. If you think we should change it, LMK

Reply:
Oh ok, that's alright then